### ISSUE TITLE
nativefier icons repo is unmaintained

### ISSUE THREAD

**Issue by DiscoveryOV** [jiahaog/nativefier#1058]
There are a lot of pull requests pending for nativefier icons.

https://github.com/jiahaog/nativefier-icons

**Comment by CoreyVidal** [comment]
Yeah, there are many outstanding pull requests, going back over a year.

I submitted the new Gmail icon 2 days ago:

https://github.com/jiahaog/nativefier-icons/pull/82
